src/GeositeFramework/js/Plugin.js

```diff
@@ -450,6 +450,9 @@ require(['use!Geosite',
                     $('.' + printCssClass).remove();
                     $printSandbox.empty();
 
+                    // add base plugin css
+                    addCss('css/print.css', 'base-plugin-print-css');
```

Comment:
`print.css` was intended to always be on the page, and not something that needs to be added dynamically. It should hold the base print rules that apply both to plugin print and map printing.